clusterNode.py

```diff
@@ -256,6 +256,12 @@ def post_boot(self, *, desired_ip_range: Optional[tuple[str, str]] = None) -> bo
         self.dynamic_ip = ips[0]
         return True
 
+    def teardown(self) -> None:
+        assert self.config.bmc is not None
+        bmc = BMC.from_bmc_config(self.config.bmc)
+        h = host.HostWithBF2(self.config.node, bmc)
```

Comment:
why HostWithBF2?

@thom311 this will break Marvell. Can you please patch Marvell to not rely on a previous run so that we can clean up properly?